### PR TITLE
Renderer media sortable

### PR DESCRIPTION
**Pull request by smeeckaert**
I added the possibility for the renderer to be sortable.

It will be automatically sortable if the relation has the key *key_through_to* set.

### CODE REVIEW

File: classes/renderer/media.php [hunk 1]
```diff
@@ -59,6 +59,13 @@ public function build()
 
         $is_multiple = isset($this->options['multiple']) ? $this->options['multiple'] : is_array($item->{$field_name});
 
+        $keyThrough = null;
+        if (!empty($relation)) {
```
Comment (jguyomard) [inline classes/renderer/media.php]: + && !empty($relation->key_through_to) ?

Comment (smeeckaert) [inline classes/renderer/media.php]: Can't do, the __isset() method is not set in Orm\Relation.

**Comment by jguyomard** [comment]
:+1:

**Comment by shaoshiva** [comment]
:+1:

**Pull request merged**